tests/backend/test_bosonic_backend.py

```diff
@@ -39,8 +41,24 @@ def test_kron_list(self):
         l2 = [3, 4, 5]
         list_compare = [3, 4, 5, 6, 8, 10]
         assert np.allclose(list_compare, bosonic.kron_list([l1, l2]))
-
-
+        
+class TestParameterChecker:
+    """Test parameter_checker function from the bosonic backend."""
+
+    def test_parameter_checker(self):
+        symbolic_param = sympy.Expr()
+        params = []
+        assert not bosonic.parameter_checker(params)
+        
```

Comment:
you probably want to pass `black` on this new additions to the file.

Reply:
Done.